framework/src/interfaces/Coupleable.C

```diff
@@ -321,6 +322,27 @@ Coupleable::getDefaultVectorValue(const std::string & var_name)
   if (default_value_it == _default_vector_value.end())
   {
     VectorVariableValue * value = new VectorVariableValue(_coupleable_max_qps, 0);
+    bool already_warned = false;
+    for (unsigned int qp = 0; qp < _coupleable_max_qps; ++qp)
+      for (unsigned int i = 0; i < LIBMESH_DIM; ++i)
+      {
+        try
+        {
+          (*value)[qp](i) = _c_parameters.defaultCoupledValue(var_name, i);
```

Comment:
Oh... so you did wrap the access of `at()`. I suppose this is ok, just not my favorite construct but valid.